etc/ukv/varobs/Radar.nl

```diff
@@ -0,0 +1,3 @@
+&VarobsControlNL
+Varfields=63,64,65,66,69,75
```

Comment:
I'm not sure I fully understand why varfield 66 is requested here but the test for it has been removed below - is that because it is used in the processing but does not need to be passed to VAR?

Reply:
This is a quirk of the Varobs structure that is used for Doppler winds. 63 is the observable of interet (radial wind), and the other ones are all metadata. OPS outputs all of these at once. Therefore I thought it made more sense to add a ctest that does this, rather than focus on the individual metadata variables. Furthermore, the ctest for variable 66 wasn't working because the obsspace was called `Radar` (and therefore expected all of the auxiliary variables to be present). There are probably ways around that but I don't think they would add much beyond the existing ctest.

Reply:
Thanks for the answer Chris, that makes sense! I'll resolve this conversation now, happy to approve